gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
```

Comment:
Do we really need to revert back to `7.1` ?

The difference between `-all` and `-bin` is that `-all` includes source code files that we might want to see when debugging some scripts.

`-bin` has only the runtime stuff needed to run Gradle.

Reply:
I don't think I changed this on purpose, something else changed this as a side effect.